.github/workflows/ci.yml

```diff
@@ -44,3 +44,21 @@ jobs:
           cd build
           cp /usr/x86_64-w64-mingw32/lib/libwinpthread-1.dll test/ || true # used only for MinGW tests, ok to fail for other cases
           ctest --output-on-failure
+  fuzzing_build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Ubuntu Build Dependencies
+        run: |
+          sudo apt update
+          sudo apt install libsdl2-dev liblzo2-dev libssl-dev gnutls-dev libgcrypt-dev mingw-w64-x86-64-dev binutils-mingw-w64-x86-64 gcc-mingw-w64-x86-64 wine clang
+      - name: Build
+        env:
+          CC: "clang"
+          LIB_FUZZING_ENGINE: "-fsanitize=fuzzer"
+          CFLAGS: "-fsanitize=address,fuzzer-no-link -DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION=1"
+        run: |
+          mkdir build
+          cd build
+          cmake ..
+          cmake --build .
```

Comment:
I see that a build is done, but where's the actual test? ;-)

Reply:
What do you want to test beyond the build ?
We can run he fuzzers for a few seconds...

But I recommend to use CIFuzz GitHub actions (which works only if project is integrated into oss-fuzz).

This is because CIFuzz will have up to date corpus to have better fuzzing

Reply:
Ah, did'nt read this comment of yours. Can you point me to some docs that explain how this machinery is supposed to work?

Reply:
Thanks!